libs/agent-sm/agent/src/main/java/org/opensearch/javaagent/Agent.java

```diff
@@ -33,6 +36,18 @@ public class Agent {
      */
     private Agent() {}
 
+    /**
+     * List of methods that are intercepted
+     */
+    public static final String[] INTERCEPTED_METHODS = new ArrayList<String>() {
```

Comment:
```suggestion
  public static final String[] INTERCEPTED_METHODS = Stream.concat(
        Stream.of("read", "open"),
        Stream.concat(FileInterceptor.MUTATING_OPERATIONS.stream(), FileInterceptor.DELETE_OPERATIONS.stream())
    ).toArray(String[]::new);

```

Reply:
It seems like sharing is causing issues, let's duplicate these constants, sorry for mislead here